### PR TITLE
Add chinese translation for quickstart page

### PR DESCRIPTION
**Pull request by qianfeiqianlan**
Hi docs team,

I do some translate work for chinese docs, for this PR I transalte the **quickstart** page.

Add the install page and using helm page was just make it not break compile.

If any issue for this PR, give me feedback please.

@flynnduism @bridgetkromhout

### CODE REVIEW

File: content/zh/docs/intro/install.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+title: "Installing Helm"
+description: "Learn how to install and get running with Helm."
+weight: 2
+aliases: ["/docs/install/"]
```
Comment (bridgetkromhout) [inline content/zh/docs/intro/install.md:5]: Please remove this "aliases" line.

File: content/zh/docs/intro/quickstart.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
+---
+title: "快速入门指南"
+description: "如何安装和开始使用Helm，包括Helm的介绍、FAQs和Helm plugins"
+weight: 1
+aliases: ["/docs/quickstart/"]
```
Comment (bridgetkromhout) [inline content/zh/docs/intro/quickstart.md:5]: Please remove this "aliases" line.

File: content/zh/docs/intro/quickstart.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+如果您在执行 `helm uninstall` 的时候提供 `--keep-history` 选项， Helm将会保存release history。
+您可以通过命令查看该release的信息
+
+```console
+$ helm status smiling-penguin
+Status: UNINSTALLED
+...
+```
+
+因为即使您卸载了它们，Helm也会记录您的release，您可以审计集群的历史记录或者取消 `helm uninstall` 操作（使用 `helm rollback` ）。
```
Comment (bridgetkromhout) [inline content/zh/docs/intro/quickstart.md:105]: Does this say that rollback cancels an uninstall command? I do not think that is accurate.

**Comment by bridgetkromhout** [comment]
Thank you for working on this translation, @qianfeiqianlan! I have added some comments. Thanks.

File: content/zh/docs/intro/quickstart.md (hunk 3 above)
Comment (qianfeiqianlan) [reply content/zh/docs/intro/quickstart.md]: Hi, I translated it with more accurate expressions. Please check it well.

**Comment by bridgetkromhout** [comment]
Hi, @qianfeiqianlan! Thank you. The translation looks good. We can merge as soon as the commit is signed.

Can you please sign the new commit correctly? Directions are here: https://github.com/helm/helm-www/pull/798/checks?check_run_id=1077732675

**Comment by qianfeiqianlan** [comment]
> Hi, @qianfeiqianlan! Thank you. The translation looks good. We can merge as soon as the commit is signed.
> 
> Can you please sign the new commit correctly? Directions are here: https://github.com/helm/helm-www/pull/798/checks?check_run_id=1077732675

Hello, @bridgetkromhout  I have signed the last commit.

**Review by bridgetkromhout — APPROVED** [review]

**Comment by bridgetkromhout** [comment]
Hi, @qianfeiqianlan! Thank you!

**Pull request merged**